specification/ai/ImageAnalysis/main.tsp

```diff
@@ -0,0 +1,36 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "./routes.tsp";
+
+using Azure.Core.Traits;
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+@service({
+  title: "Image Analysis",
+})
+@versioned(ImageAnalysis.Versions)
+@useAuth(ApiKeyAuth<ApiKeyLocation.header, "Ocp-Apim-Subscription-Key">)
+@useAuth(BearerAuth)
+@server(
+  "{endpoint}/computervision",
+  "Analyze an image to generate a description of its content, detect text, objects and more.",
+  {
+    @doc("""
```

Comment:
nit: You can now use JSDoc comment style `/** xxx */` in place of `@doc`. This allows better IDE visibility since comment block is dark green.

In VSCode, you could use regex search of `@doc\("{1,3}((.|\n)*?)"{1,3}\)` and replace with `/** $1 */` for a global search and replace.

Reply:
Can be for future reference...

Reply:
That's a good tip, thank you! If we need another minor TypeSpec update before SDK release, I will make sure to include this change. @rhurey for visibility.